src/library/parsercsv.cpp

```diff
@@ -180,12 +180,22 @@ bool ParserCsv::writeCSVFile(const QString &file_str, BaseSqlTableModel* pPlayli
                 first = false;
             }
             out << "\"";
-            QString field = pPlaylistTableModel->data(pPlaylistTableModel->index(j,i)).toString();
-            if (useRelativePath &&
-                    i ==
-                            pPlaylistTableModel->fieldIndex(ColumnCache::
-                                            COLUMN_TRACKLOCATIONSTABLE_LOCATION)) {
-                field = base_dir.relativeFilePath(field);
+            QString field;
+            if (i == pPlaylistTableModel->fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_BPM)) {
+                // Use ToolTip Role for full precision.
```

Comment:
This is a bad design with implicit assumptions that is supposed to break at any time. No one will notice that the CSV depends on the tooltip role when changing the precision of or output of the tooltip role once again. It would be reasonable to display "xxx.xxx bpm" as a tooltip which would break the CSV export.

Reply:
Do you have an idea how to fix this without a mayor refactoring.

While I agree that this is a bad design, I have no interest to spend too much time on this. 

At least it will not break on future changes, because it is text export.

I would than better revert the rounding to one decimal place in the library table. 

The issue is, that we have original missed the important use case to instantly distinguish  1/12 beats from others.

Reply:
As an ugly workaround you have to describe this dependency as comments. Bidirectional, i.e. on both sides!

Reply:
Moreover the CSV export should use "." as a decimal separator while the tooltip should ideally use the decimal separator from the localization settings. Completely different use case that deserves two separate formatting functions.

Reply:
As far as I remember, we use in Mixxx the decimal dot even in German local. 
This was decided, because a decimal coma looks foreign in a DJ Application.

But anyway the scope of this PR is to fix a regression. I will go with the warning comments.

Reply:
Conceptually these are different things and should not be mixed. But I agree that we don't need to fix it here and now. Nevertheless he have to record those issues when we stumble over them.

Reply:
I think we should have a export role with a mixxx wide constant that is used for exporting data and used from all exporters.

Reply:
OK, done.